host: MacBook-Pro-von-Frederic.local
Building DAG of jobs...
Using shell: /opt/local/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job stats:
job             count
------------  -------
combine_exps        1
complete_cx         1
complete_exp        1
total               3

Select jobs to execute...
Execute 1 jobs...

[Sat Dec  7 17:14:09 2024]
localrule complete_cx:
    input: results/__cythonized__
    output: results/synthetic/complete/n=50 r=0.5 re=True d=False/cx/e=False p=False l=10 s=1000/run/cx_0.csv
    jobid: 2
    reason: Missing output files: results/synthetic/complete/n=50 r=0.5 re=True d=False/cx/e=False p=False l=10 s=1000/run/cx_0.csv
    wildcards: n_nodes=50, neg_edge_prob=0.5, neg_edge_dist_exact=True, directed=False, exact=False, parallel=False, max_length=10, n_samples=1000, run=0
    resources: tmpdir=/var/folders/dj/14c985vd1ng7gf0vvw_ytrpw0000gn/T

Terminating processes on user request, this might take some time.
RuleException:
CalledProcessError in file /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/workflow/Snakefile, line 136:
Command 'set -euo pipefail;  /Users/fredericbusch/miniforge3/envs/py-raccoon/bin/python3.12 /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/.snakemake/scripts/tmpk48wg6gy.experiment.py' returned non-zero exit status 1.
[Sat Dec  7 17:14:15 2024]
Error in rule complete_cx:
    jobid: 2
    input: results/__cythonized__
    output: results/synthetic/complete/n=50 r=0.5 re=True d=False/cx/e=False p=False l=10 s=1000/run/cx_0.csv

Complete log: .snakemake/log/2024-12-07T171409.492763.snakemake.log
WorkflowError:
At least one job did not complete successfully.
